spaces/toncommunity/index.json

```
@@ -0,0 +1,16 @@
+{
+  "key": "ton",
```

The key of your space must also be the folder name and skin class name.
```suggestion
  "key": "toncommunity",
```
Or rename your space directory from `toncommunity` to `ton`